### ISSUE TITLE
Won't run at all

### ISSUE THREAD

**Issue by EvilCaptKirk** [DEROFDN/Engram#9]
Latest wallet version (0.1.2?) on Windows 10 Pro. Downloaded and extracted the binary, try to run it - either the window appears for a split second, long enough to see it's outline, then immediately closes (most common), or it appears but hangs before any content can load (only happened once, and not on the first run attempt).

I've added an exception for it to antivirus, tried running as Administrator and various compatibility modes - no joy. No error messages and no error log that I've been able to find. 

Appreciate any help.

**Comment by DankFC** [comment]
Engram is a portable app, so it needs to have write permissions in order to function. Make sure of the following:

1. The app was extracted before opening.
2. The app has write permissions where it is stored.
3. Check your settings to make sure it is connecting to a node, or running a node itself. You can click the 'restore defaults' button at the bottom of the setting screen and restart Engram to correct connection issues.

**Comment by EvilCaptKirk** [comment]
I did extract the binary before attempting to run it, and it does have write permissions. I've also tried running it as Administrator, but that does not help.

It creates a couple of folders named 'datashards' and 'mainnet' when it runs, but it always crashes or otherwise fails to create an application window - always quietly, with no error messages. As such, there's no way for me to check for node connection.

Any suggestions on troubleshooting steps to get some meaningful error messages?
